docs/advanced/html.md

```diff
@@ -49,33 +49,62 @@ A more verbose but "stable" approach is to use the `:class:` keyword argument wh
 :::
 
 Currently, using the `{admonition}` directive with a title creates a CSS class based on the title of the admonition.
-For example, an admonition title of `Here's my title` will result in a class name of `admonition-heres-my-title`.
+For example, an admonition title of `Here's my title` will result in a class name of `admonition-here-s-my-title`.
+
+On the other hand, by using the `:class:` keyword argument, it will create a class with the keyword previously chosen.
+For example, a custom class defined as `my-custom-class` will result in a class named as `my-custom-class`.
 
 You can leverage this pattern to quickly create custom admonitions.
 For example, create a `myadmonitions.css` under `mybook/_static` with the following CSS:
 
+#### Using the `{admonition}` directive with a title
+
 ```css
-.admonition-extra-credit {
+div.admonition-extra-credit {
```

Comment:
why add the `div`?  I think we may not want to do this because different versions of `sphinx` make this a `div` vs. an `aside`

Reply:
That's because the .css files from sphinx would overlap the custom .css, since they have the .div in front and thus being more specific.

@brownsarahm opened an issue related to it (#1558) and, after my PR, noticed that it also solved her issue.